monorepo/website/src/pages/news/2025-04-10-spring-update.mdx

```diff
@@ -0,0 +1,60 @@
+- A new "Report an Issue" button on sequence pages allows users to flag potential concerns more easily than ever ([Link](https://github.com/loculus-project/loculus/pull/3681))
+ ![screenshot showing the 'Create Github Issue' button at the bottom of a sequence information page](/gifs/news/2025-04-10-spring-update/reportIssue.jpg)
+- And if you just have one sequence to submit, that’s now simple too, as users can now submit individual sequences via an easy-to-use web form ([Link](https://github.com/loculus-project/loculus/pull/3710))
+- Thanks to user feedback, a display issue with the DUT column was identified and quickly resolved ([Link](https://github.com/loculus-project/loculus/pull/3604))
+- Seeing what filters you’ve applied to your search is now straightforward, as they appear above the search results, improving visibility and usability ([Link](https://github.com/loculus-project/loculus/pull/3777))
+ ![GIF showing how search criteria is displayed at the top of the page](/gifs/news/2025-04-10-spring-update/filtersOnSearch.gif)
+- It’s now possible to download unaligned sequences with the country and date in the fasta-header (select ‘Display name’ when downloading) ([Link](https://github.com/loculus-project/loculus/pull/3814))
+- Revising sequences is now easier thanks to file uploads for sequences ([Link](https://github.com/loculus-project/loculus/pull/3812)) and drop-down country-selection ([Link](https://github.com/loculus-project/loculus/pull/3842))
+
+
+### Meetings, Minutes, and Resolutions:
+- Remember that you can always check our [minutes from General Assemblies and Executive Board meetings, as well as Executive Board resolutions](https://pathoplexus.org/about/governance/minutes)
+
+### Past talks:
+- Emma Hodcroft spoke about Pathoplexus and Loculus at the [Center for Pathogen Bioinformatics Launch](https://www.sib.swiss/news/the-centre-for-pathogen-bioinformatics-epidemic-preparedness-beyond-covid-19) (23 Jan, Bern, CH)
+- Emma Hodcroft spoke about Pathoplexus at “Cloud Computing Africa” (26 Mar, The Gambia (remote))
+- Theo Sanderson, Anderson Brito, Anna Parker, and Danny Park attended [GA4GH Connect 2025](https://www.ga4gh.org/event/april-connect-2025/) to coordinate with the global genomics community on standards and connectivity (1-4 April 2025, Boston, USA)
+- Cornelius Roemer spoke about Pathoplexus at [Emerging viruses and new vaccines - from research to public health](https://www.unige.ch/emerging-virus-symposium/) (7-9 April 2025, Geneva, CH)
+
+### Upcoming talks:
+Hear more about Pathoplexus & Loculus at these upcoming meetings!
+- [Dynamics & Evolution of Human Viruses](https://dynamicsevolution.org/event/5/) - 6-9 May, Asnières-sur-Oise, France
+- [Applied Bioinformatics & Public Health Microbiology (ABPHM)](https://coursesandconferences.wellcomeconnectingscience.org/event/applied-bioinformatics-public-health-microbiology-20250521/) - 21-23 May, Hinxton, UK
```

Comment:
Should we add the names here as well? In the past, we always mentioned the corresponding people.

```suggestion
- Emma Hodcroft and Chaoran Chen will present at [Dynamics & Evolution of Human Viruses](https://dynamicsevolution.org/event/5/) - 6-9 May, Asnières-sur-Oise, France
- Chaoran Chen will present at [Applied Bioinformatics & Public Health Microbiology (ABPHM)](https://coursesandconferences.wellcomeconnectingscience.org/event/applied-bioinformatics-public-health-microbiology-20250521/) - 21-23 May, Hinxton, UK
```